src/python/espressomd/observables.py

```diff
@@ -69,6 +70,18 @@ def bin_centers(self):
         return np.array(list(itertools.product(*edges))).reshape(shape)
 
 
+class CylindricalProfileObservable(ProfileObservable):
+    """
+    Base class for observables that work with cylinder coordinates
+    """
+
+    def __init__(
+            self, trafo_params=CylindricalTransformationParameters(), **kwargs):
+        # Provide default transformation parameters if not user-provided
+        kwargs['trafo_params'] = trafo_params
```

Comment:
here and elsewhere, did you mean to write `transform_params` instead of `trafo_params` for the optional argument?

Reply:
I was thinking that transform_params sounds a bit like the params get transformed

Reply:
no, I think it's fine 😊

Reply:
I agree, `transform_params` is transparent and the docstring is clear about its role

Reply:
ok